### PR TITLE
relax ttl validation to allow passive cookies

### PR DESCRIPTION
**Pull request by ramaraochavali**
Fixes https://github.com/istio/istio/issues/46922. Will update API docs after this is merged.

- [ ] Ambient
- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Dual Stack
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by howardjohn — COMMENTED** [review]
Do we have any tests for this?

**Comment by ramaraochavali** [comment]
Let me check - I have n't checked the integration tests.

File: pkg/config/validation/validation.go [hunk 1]
```diff
@@ -1448,13 +1448,8 @@ func validateLoadBalancer(settings *networking.LoadBalancerSettings, outlier *ne
...
 	if consistentHash != nil {
 		httpCookie := consistentHash.GetHttpCookie()
-		if httpCookie != nil {
-			if httpCookie.Name == "" {
-				errs = appendValidation(errs, fmt.Errorf("name required for HttpCookie"))
-			}
-			if httpCookie.Ttl == nil {
-				errs = appendValidation(errs, fmt.Errorf("ttl required for HttpCookie"))
-			}
+		if httpCookie.GetName() == "" {
```
Comment (hzxuzhonghu) [inline pkg/config/validation/validation.go:1451]: Not right, revert to `httpCookie != nil`

**Review by hzxuzhonghu — APPROVED** [review]
We should have a test coverage on consistent hash

**Comment by ramaraochavali** [comment]
Oh. I am adding test case. I realized the test does not exist for http cookie. Let me add those tests in a separate PR

**Pull request merged**